pkg/controllers/state/suite_test.go

```diff
@@ -636,6 +677,19 @@ var _ = Describe("Pod Anti-Affinity", func() {
 
 })
 
+func ExpectMetricLabel(mf *prometheus.MetricFamily, name string, value string) {
```

Comment:
Where is this being used?

Reply:
I believe thats leftover from when the pod metrics were in cluster state. I'll remove it.